src/ts4.1/index.d.ts

```diff
@@ -228,5 +228,5 @@ export interface TranslationProps<N extends Namespace = DefaultNamespace> {
 }
 
 export function Translation<N extends Namespace = DefaultNamespace>(
-  props: TranslationProps<N>,
+  props: TranslationProps<N> & { useSuspense?: boolean },
```

Comment:
Hey @depoulo, thanks for your contribution. 
`src/index.d.ts` is right, but here we should place `useSuspense` under TranslationProps.